app/src/main/java/com/futurice/freesound/feature/home/HomeFragment.java

```diff
@@ -0,0 +1,158 @@
+import com.futurice.freesound.inject.fragment.BaseFragmentModule;
+import com.futurice.freesound.utils.Preconditions;
+import com.futurice.freesound.viewmodel.Binder;
+import com.squareup.picasso.Picasso;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import javax.inject.Inject;
+
+import butterknife.BindView;
+import butterknife.Unbinder;
+import io.reactivex.disposables.CompositeDisposable;
+import io.reactivex.schedulers.Schedulers;
+import polanski.option.AtomicOption;
+import timber.log.Timber;
+
+import static butterknife.ButterKnife.bind;
+import static com.futurice.freesound.utils.Preconditions.get;
+import static io.reactivex.android.schedulers.AndroidSchedulers.mainThread;
+
+public final class HomeFragment extends BindingBaseFragment<HomeFragmentComponent> {
+
+    @Nullable
+    @Inject
+    HomeFragmentViewModel homeFragmentViewModel;
+
+    @Nullable
+    @Inject
+    Picasso picasso;
+
+    @Nullable
+    @BindView(R.id.avatar_image)
+    ImageView avatarImage;
+
+    @Nullable
+    @BindView(R.id.username)
+    TextView userName;
+
+    @Nullable
+    @BindView(R.id.about)
+    TextView about;
+
+    @NonNull
+    private final AtomicOption<Unbinder> unbinder = new AtomicOption<>();
+
+    @NonNull
+    private final Binder binder = new Binder() {
+
+        @Override
+        public void bind(@NonNull final CompositeDisposable d) {
+            Preconditions.checkNotNull(homeFragmentViewModel);
+
+            d.add(homeFragmentViewModel.getImage()
+                                       .subscribeOn(Schedulers.computation())
+                                       .observeOn(mainThread())
+                                       .subscribe(it -> get(picasso).load(it)
+                                                                    .into(avatarImage),
+                                                  e -> Timber.e(e, "Error setting image")));
+
+            d.add(homeFragmentViewModel.getUserName()
+                                       .subscribeOn(Schedulers.computation())
+                                       .observeOn(mainThread())
+                                       .subscribe(it -> get(userName).setText(it),
+                                                  e -> Timber.e(e, "Error setting user")));
+
+            d.add(homeFragmentViewModel.getAbout()
+                                       .subscribeOn(Schedulers.computation())
+                                       .observeOn(mainThread())
+                                       .subscribe(it -> get(about).setText(it),
+                                                  e -> Timber.e(e, "Error setting about")));
+        }
+
+        @Override
+        public void unbind() {
+            // Nothing to do here
```

Comment:
I think we do care because conceivably if it's inflight when this is unsubscribed it could still attempt to update the view. Perhaps it might not strictly leak because I believe Picasso keeps the view (a Picasso `Target`) as a weak references, but if we don't cancel then we are still holding onto that request for longer than we need to. 

Take a look at `SoundItemViewHolder` - it its `unbind` we cancel the request for the waveform image when the RecyclerView item is being recycled/detached. For the most part, that is done for the same reasons we would want to do it here.